### PR TITLE
Code Review v1.0

### PR DESCRIPTION
**Pull request by calebfavor**

### CODE REVIEW

File: src/assets/sass/variables.scss [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+$reactionYellowColor: #FFC841;
+$reactionOrangeColor: #E25822;
+$reactionRedColor: #F71B26;
+$reactionGreenColor: #00BC75;
+$reactionBlueColor: #0B76DB;
+
+$teamColor: #000;
+$edgeColor: #0B76DB;
+$lifetimeColor: #07B3FF;
+$coachColor: #FAA300;
```
Comment (miguelfondeur) [inline src/assets/sass/variables.scss]: When we bring in Stylesora, we can use the tailwind color variables. I've added a lot of the colors from Musora's color palette. But we can always extend the tailwind.config to add more.

**Review by miguelfondeur — APPROVED** [review]
We  discussed refactoring large template files and inheriting the Stylesora.css from each website where Chatsora will live. Those are optimizations. For now this looks good!

File: src/assets/js/data/emoji.json [hunk 2]
```diff
@@ -0,0 +1,138 @@
+{
+    "frequent": [
```
Comment (trentharms) [inline src/assets/js/data/emoji.json]: I've noticed some platforms limit the amount of emojis you can react with- do you think we should funnel down to like 5 or 6 to start with?

check out iMessage and Facebook Messenger for some examples

**Review by trentharms — COMMENTED** [review]
looks great dude! I cant wait to see it on production!

**Pull request closed without merging**